docs/mine/connectivity.md

```diff
@@ -86,3 +82,58 @@ Below is an example `~/.lotusminer/config.toml` configuration file in which the
 In the above example, port `10240` is forwarded to `<internal-miner-host-ip>:5472`.
 
 It is possible to verify that the port is listening by using telnet (eg: `telnet 1.2.3.4 10240`. `nc` is also sufficient.) If successful, a plaintext `/multistream/1.0.0` line will be within the response.
+
+## Setting up a Relay endpoint
+
+If you find yourself in a situation in which you set a public IP address on your machine (e.g. Residential Networks) and/or you do not control the NAT/Firewall that your device is behind (e.g. Entreprise Networks and other Firewalls), there is an alternative solution for you. Setting up a Relay endpoint so that your miner can relay its internet traffic through an external, publicly diable endpoint.
+
+Below you can find multiple ways in how to achieve this.
+
+Note: Remember that libp2p (the underlaying network stack of the Filecoin miner) will be listening on multiple addresses simultaneously. What this means is that adding a Relay endpoint is not a tradeoff but an advantage, as it will be used for last resort when direct connectivity can't be achieved.
+
+### libp2p Relay
+
+The [libp2p Circuit Relay (docs here)](https://docs.libp2p.io/concepts/circuit-relay/) is just a standard libp2p node that offers a service to any other Relay node to route their traffic through it. With it, you can deploy a libp2p Circuit Relay in a machine with a Public IP address (e.g. a standard Cloud Provider) and by adding a libp2p relay multiaddr to your miner node, you will instruct the miner to route all of its traffic through the libp2p Relay.
+
+You can learn:
+- How it works [here](https://docs.libp2p.io/concepts/circuit-relay)
+- How to write a simple program that does the relay [here](https://github.com/libp2p/go-libp2p-examples/blob/master/relay/main.go)
+
```

Comment:
@gmas, @thattommyhall I was asking for a libp2p Relay deployment example so that I could add it here.

Reply:
IIRC Juan suggested we shouldn't provide an existing deployment